configmodel/models/scanners/zap.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable=invalid-name
+import logging
+from dataclasses import dataclass
+from dataclasses import field
+from enum import Enum
+from typing import List
+from typing import Optional
+from typing import Union
+
+
+class ImportUrlsFromFileType(str, Enum):
+    HAR = "har"
+    MODSEC2 = "modsec2"
+    URL = "url"
+    ZAP_MESSAGES = "zap_messages"
+
+
+@dataclass
+class ZapApis:
+    apiUrl: Optional[str] = None
+    apiFile: Optional[str] = None
+
+    def __post_init__(self):
+        has_url = self.apiUrl is not None and self.apiUrl != ""
+        has_file = self.apiFile is not None and self.apiFile != ""
+
+        if not (has_url or has_file):
+            raise ValueError("No apiUrl or apiFile is defined in the config, in apiScan.apis")
+
+
+@dataclass
+class ZapApiScan:
+    apis: ZapApis
+
+
+@dataclass
+class ZapImportUrlsFromFile:
+    fileName: str
+    type: ImportUrlsFromFileType = ImportUrlsFromFileType.URL
+
+
+@dataclass
+class ZapGraphql:
+    endpoint: Optional[str] = None  # pylint: disable=invalid-name
```

Comment:
I suppose you don't need these pylint annotations anymore since there's one added to the top of this file.

Reply:
Done